vars/perfRegressionParallelPipeline.groovy

```diff
@@ -110,6 +110,26 @@ def call(Map pipelineParams) {
                    description: 'jobs to compare performance results with, for example if running in staging, '
                                 + 'we still can compare with official jobs',
                    name: 'perf_extra_jobs_to_compare')
+
+            // NOTE: Optional parameters for BYO ScyllaDB stage
+            string(defaultValue: '',
+                   description: (
+                       'Custom "scylladb" repo to use. Leave empty if byo is not needed. ' +
+                       'If set then it must be proper GH repo. Example: git@github.com:personal-username/scylla.git\n' +
+                       'and, in case of an "rolling upgrade", need to define "base_versions" param explicitly.'),
+                   name: 'byo_scylla_repo')
+            string(defaultValue: '',
+                   description: 'Branch of the custom "scylladb" repo. Leave empty if byo is not needed.',
+                   name: 'byo_scylla_branch')
+            string(defaultValue: '/scylla-master/byo/byo_build_tests_dtest',
```

Comment:
@soyacz which default will be more useful here?

Reply:
I think this default is o.k.

I would guess that most time one is building from something based on master.

There's no easy way to figure this one out, who created the scylla fork/branch  should know which base branch he used